py_ms_cognitive/__init__.py

```diff
@@ -1,4 +1,4 @@
-from py_ms_cognitive_search.py_ms_cognitive_web_search import PyMsCognitiveWebSearch
-from py_ms_cognitive_search.py_ms_cognitive_news_search import PyMsCognitiveNewsSearch
-from py_ms_cognitive_search.py_ms_cognitive_video_search import PyMsCognitiveVideoSearch
-from py_ms_cognitive_search.py_ms_cognitive_image_search import PyMsCognitiveImageSearch
\ No newline at end of file
+from .py_ms_cognitive_search.py_ms_cognitive_web_search import PyMsCognitiveWebSearch
```

Comment:
I'm absolutely fine with how things are, but wouldn't pep-8 suggest not using relative imports (e.g. `from  py_ms_cognitive.` instead of `from .`)?